content/documentation/edge/installation-guide/index.adoc

```diff
@@ -99,35 +99,35 @@ The version of Kiali packaged with Istio or Maistra may not contain the most rec
+The next few sections describe the different installation methods available for installing the Kiali Operator. Depending on the installation method you use, the operator may then install Kiali. Before you install Kiali, you will want to know what authentication strategy Kiali should use (this determines how users are to log into Kiali). See link:#_login_options[Login Options] for more on the `login`, `anonymous`, `ldap`, `openshift`, `token` and `openid` authentication strategies.
 
-icon:bullhorn[size=1x]{nbsp} It is only necessary to install the Kiali Operator one time. After the operator is installed you need only create or edit the Kiali CR (see link:#_create_or_edit_the_kiali_cr[Create or Edit the Kiali CR]). The Kiali config map will be managed by the Kiali Operator and should *not* be manually edited. There is no need to again perform one of the bash installations below.
+icon:bullhorn[size=1x]{nbsp} It is only necessary to install the Kiali Operator one time. After the operator is installed you need only create or edit the Kiali CR (see link:#_create_or_edit_the_kiali_cr[Create or Edit the Kiali CR]). The Kiali ConfigMap will be managed by the Kiali Operator and should *not* be manually edited. There is no need to again perform one of the installations below once the Kiali operator is installed and running.
 
 
 === Quick Install
 
-This option installs the Kiali Operator and the Kiali CR with default options. If you are running with an https://github.com/istio/istio/releases[upstream Istio release], you will probably want to pass in _--accessible_namespaces="**"_ since that is the typical way Kiali is expected to run within an upstream Istio environment.
-
-
-When the Kiali CR is created it triggers the operator to then install Kiali. Kiali is able to access existing namespaces, but it required a CR update to access future namespaces. See link:#_namespace_management[Namespace Management] for more information. Run the install script via this command:
+This option installs the Kiali Operator and the Kiali CR with default options. If you are running with an https://github.com/istio/istio/releases[upstream Istio release], you will probably want to pass in _--accessible-namespaces="**"_ since that is the typical way Kiali is expected to run within an upstream Istio environment.
 
 [source,bash]
 ----
-  bash <(curl -L https://kiali.io/getLatestKialiOperator) --accessible-namespaces '**'
+  bash <(curl -L https://kiali.io/getLatestKialiOperator) --accessible-namespaces="**"
 ----
 
+In this case, Kiali can access currently existing namespaces as well as future namespaces that may be created later. See link:#_namespace_management"[Namespace Management] for more information if you want to change that behavior.
```

Comment:
In this case...,  -->  This option gives Kiali access to existing namespaces as well as namespaces created later.

Reply:
will be done